src/main/java/slimeknights/tconstruct/library/Util.java

```diff
@@ -222,4 +222,18 @@ public static String temperatureString(int temperature, boolean celsius) {
     }
     return translateFormatted("gui.general.temperature.kelvin", temperature);
   }
+
+  private static int minFluidHeight = 3;
```

Comment:
This should not be stored inside the util class. Utils are stateless. You want to store it in the Config class, where all the other configurations are also stored.

Reply:
The reason I originally suggested util for this was to keep config access out of lib. Is there a preferred location so lib does not rely on lib?

Reply:
We only have 1 place where it's accessed, we don't really need a Util for it. Should be fine to just access the config there.